install/configure.sh

```diff
@@ -6,6 +6,30 @@
 
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+echo ROOT=${ROOT}
+
+if [ ! -d "${ROOT}/.clean_" ]
+then
+    echo "Initializing CWS directory... "
+    mkdir ${ROOT}/.clean_
+    # Save clean CWS before configure.sh is run and directory is modified
+    cp -r ${ROOT}/* ${ROOT}/.clean_ && rm -rf ${ROOT}/.clean_/bpmn
+else
+    echo "Re-running CWS Config, backing up previous configuration(s). Refer to .backups_ folder."
```

Comment:
Change this to:
Re-running CWS Config, backing up current configuration.  Refer to .backups_ folder for all backups.

Reply:
Re-run CWS statement changed